buildkite/access_tokens.go

```diff
@@ -0,0 +1,62 @@
+package buildkite
+
+import "fmt"
+
+type AccessTokensService struct {
+	client *Client
+}
+
+type AccessToken struct {
+	UUID   *string   `json:"uuid,omitempty" yaml:"uuid,omitempty"`
+	Scopes *[]string `json:"scopes,omitempty" yaml:"scopes,omitempty"`
+}
+
+// Get gets the current token which was used to authenticate the request
+//
+// buildkite API docs: https://buildkite.com/docs/rest-api/access-token
+func (ats *AccessTokensService) Get() (*AccessToken, *Response, error) {
+
+	var u string
+
+	u = fmt.Sprintf("v2/access-token")
+
+	req, err := ats.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	accessToken := new(AccessToken)
+
+	resp, err := ats.client.Do(req, accessToken)
+	if err != nil {
+		fmt.Print(err)
+		return nil, resp, err
+	}
+
+	return accessToken, resp, nil
+}
+
+// Revokes the current token which was used to authenticate the request
+//
+// buildkite API docs: https://buildkite.com/docs/rest-api/access-token
+func (ats *AccessTokensService) Revoke() (*AccessToken, *Response, error) {
```

Comment:
When I ran this, it returned an error:

```
➜  go-buildkite git:(alam0rt/master) ✗ go run test.go token=$JJ_BK_TOKEN 
EOF2021/07/26 16:32:54 json encode failed: EOF
exit status 1
```

I think the issue is that the API is returning a 204 there [revoke-the-current-token](https://buildkite.com/docs/apis/rest-api/access-token#revoke-the-current-token) and the JSON decoder is complaining [here](https://github.com/buildkite/go-buildkite/blob/master/buildkite/buildkite.go#L273)?

Since it's a Delete call, maybe it would be better to do something similar to [delete-an-agent](https://github.com/buildkite/go-buildkite/blob/master/buildkite/agents.go#L127)?

I quickly played around and came up with something like:
```suggestion
func (ats *AccessTokensService) Revoke() (*Response, error) {

    var u string

    u = fmt.Sprintf("v2/access-token")

    req, err := ats.client.NewRequest("DELETE", u, nil)
    if err != nil {
        return nil, err
    }

    resp, err := ats.client.Do(req, nil)
    if err != nil {
        return resp, err
    }

    return resp, nil
}
```

Reply:
Have implemented this suggestion, thanks :)